js/helpers.js

```diff
@@ -716,13 +716,18 @@ Helpers.loadTemplate = function (name, params) {
  */
 Helpers.loadTemplate.cache = {
     "fixed_book_frame": '<div id="fixed-book-frame" class="clearfix book-frame fixed-book-frame"></div>',
-
-    "single_page_frame": '<div><div id="scaler"><iframe allowfullscreen="allowfullscreen" scrolling="no" class="iframe-fixed"></iframe></div></div>',
+    "single_page_frame": '<div><div id="scaler"><iframe enable-annotation allowfullscreen="allowfullscreen" scrolling="no" class="iframe-fixed"></iframe></div></div>',
```

Comment:
Can we please use the XHTML convention for no-value "boolean" attributes, i.e. `enable-annotation="enable-annotation"` (to be consistent with our existing `allowfullscreen="allowfullscreen"` attribute)